src/coreclr/vm/interpexec.h

```diff
@@ -61,6 +61,18 @@ struct InterpThreadContext
     ~InterpThreadContext();
 };
 
-void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFrame *pFrame, InterpThreadContext *pThreadContext);
+//
+// This overloaded function serves two purposes:
+//
+// It could be used to call a function, where throwable, pHandler and handlerFrame is null, or
+// It could be used to call a funclet, where throwable should be the exception object, ip is the bytecode offset of the handler,
+//
+// In the case of calling a function, it will return nullptr
+// In the case of catch, it will return the bytecode address where the execution should resume, or
+// In the case of filter, it will return the decision to either execute on the current handler or continue searching for another handler.
+//
+DWORD_PTR ExecuteInterpretedCode(TransitionBlock* pTransitionBlock, TADDR byteCodeAddr, OBJECTREF throwable, void* pHandler, InterpMethodContextFrame* handlerFrame, bool isFilter);
```

Comment:
I don't think we should be having so many arguments for `ExecuteInterpretedCode`. For the invoke from EH path it seems cleaner and more perf efficient to have a single struct argument that contains all relevant information necessary for this uncommon path. Then, the typical method call would pass a simple null in this place.